torch/fx/experimental/proxy_tensor.py

```diff
@@ -562,6 +562,11 @@ def wrapped(*proxies):
             lambda t: get_proxy_slot(t, tracer, t, lambda x: x.proxy),
             out
         )
+        out = pytree.tree_map_only(
+            torch.ScriptObject,
+            lambda t: get_proxy_slot(t, tracer, t, lambda x: x),
+            out
+        )
```

Comment:
The output is a plain script object, we've tracked the script object at input and assoicated it with a proxy. This is just to retrieve its corresponding proxy. So that the `create_arg` of output node is able to connect its arguments with the correct proxy instead of creating a new one.